homeassistant/bootstrap.py

```diff
@@ -62,16 +61,13 @@ def setup_component(hass: core.HomeAssistant, domain: str,
 def _handle_requirements(hass: core.HomeAssistant, component,
                          name: str) -> bool:
     """Install the requirements for a component."""
-    if hass.config.skip_pip or not hasattr(component, 'REQUIREMENTS'):
+    if hass.config.skip_pip:
         return True
 
-    for req in component.REQUIREMENTS:
```

Comment:
Ok, instead of `util.util` I will rather move both `handle_requirements` and `mount_lib` back to `bootstrap` and just make sure we can call them from script